stable/pomerium/values.yaml

```diff
@@ -17,8 +21,8 @@ config:
   policy: {}
 
 authenticate:
-  # fullnameOverride: authenticate
-  # nameOverride: authenticate
+  fullnameOverride: authenticate
+  nameOverride: authenticate
```

Comment:
that's unfortunate, I think we don't want those set by default…

Reply:
Set with empty strings instead now.